### PR TITLE
Added Google Analytics tracking support

### LINKED ISSUE #509: Add Google Analytics

### PR DESCRIPTION
**Pull request by carsongee**
Closes #509

### CODE REVIEW

File: ui/templates/base.html [hunk 1]
```diff
@@ -120,5 +120,14 @@
...
     {% endblock %}
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new
+      Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+      ga('create', '{{ settings.GOOGLE_ANALYTICS_ID }}', 'auto');
+      ga('send', 'pageview');
+    </script>
```
Comment (bdero) [inline ui/templates/base.html:132]: Nit: Should this be maybe placed into a block that only renders when the `GOOGLE_ANALYTICS_ID` setting is set?

**Comment by bdero** [comment]
Other than the nit, this looks good to me. :oncoming_police_car:

File: ui/templates/base.html (hunk 1 above)
Comment (carsongee) [inline ui/templates/base.html:132]: Ooh, fancy!

Comment (carsongee) [inline ui/templates/base.html:132]: All branched and tested

**Comment by carsongee** [comment]
You mind rechecking?

**Comment by bdero** [comment]
Sorry about the delay, this looks nice. :necktie:

**Pull request merged**